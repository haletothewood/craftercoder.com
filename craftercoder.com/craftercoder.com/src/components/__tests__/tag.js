import React from "react"
import renderer from "react-test-renderer"

import Tag from "../tag"

describe("Tag", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Tag tag="Test Tag" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
