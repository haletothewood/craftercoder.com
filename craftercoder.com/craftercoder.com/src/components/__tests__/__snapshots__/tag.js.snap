// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag renders correctly 1`] = `
<a
  href="/tags/test-tag/"
>
  Test Tag
</a>
`;
